functions/applications.py

```
@@ -167,6 +167,53 @@ def create_application_request(
+            ).first()
+            pending_application = conn.execute(
+                select(application_requests_table.c.id).where(
+                    application_requests_table.c.personnummer_hash == personnummer_hash,
+                    application_requests_table.c.account_type == "standard",
+                    application_requests_table.c.status == "pending",
+                )
+            ).first()
+            if existing_user or pending_user or pending_application:
+                raise ValueError(
+                    "Det finns redan ett standardkonto med detta personnummer."
+                )
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)